cmd/collector/main_test.go

```diff
@@ -69,12 +74,94 @@ func TestCheckManagerConfigNoFile(t *testing.T) {
 	require.Equal(t, expected, actual)
 }
 
-func TestCheckManagerConfig(t *testing.T) {
-	tmpdir := t.TempDir()
-	manager := filepath.Join(tmpdir, "manager.yaml")
+func TestManagerConfigNoAgentIDWillSet(t *testing.T) {
+	tmpDir := t.TempDir()
+	defer os.RemoveAll(tmpDir)
```

Comment:
This isn't needed as [t.TempDir()](https://pkg.go.dev/testing#T.TempDir) automatically cleans up.